### PR TITLE
new config, change votes ordering, black linting

### PR DESCRIPTION
**Pull request by Shubhaankar-Sharma**

### CODE REVIEW

**Review by takos22 — CHANGES_REQUESTED** [review]
I only asked to format the one file you were changing, not the whole project. Reformatting the whole project at once will mess up git diffs.

File: config.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import dotenv
+import os
+import urllib.parse as urlparse
+
+if os.path.isfile(".env"):
```
Comment (takos22) [inline config.py:5]: You could just do `dotenv.load_dotenv()` so it finds the env files on its own

File: config.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+}
+
+ALLOWED_HOSTS = [os.environ.get("ALLOWED_HOST")]
+
+# Discord stuff
+TOKEN: str = os.environ.get("TOKEN")  # > https://discord.com/developers/applications
+LOG_WEBHOOK = os.environ.get(
+    "LOG_WEBHOOK"
+)  # create webhook in the server for a channel and paste here
```
Comment (takos22) [inline config.py:34]: Put info about config values in the readme instead

**Review by takos22 — COMMENTED** [review]
Some small changes but lgtm

File: config.py (hunk 1 above)
Comment (Shubhaankar-Sharma) [reply config.py:5]: @takos22 what happens if it doesnt file any file?

Comment (Shubhaankar-Sharma) [reply config.py:5]: when we use it in kubernetes we wont have a .env file we would just straight up have those in env vars

File: config.py (hunk 2 above)
Comment (Shubhaankar-Sharma) [reply config.py:34]: feel free to do it :D

File: config.py (hunk 1 above)
Comment (takos22) [reply config.py:5]: > @takos22 what happens if it doesnt file any file?

It wouldn't load anything and might raise a KeyError if you try to access them

Comment (takos22) [reply config.py:5]: But in our case it doesn't change anything

**Comment by Shubhaankar-Sharma** [comment]
Done :+1:

**Pull request merged**